### DISCUSSION TITLE
Why isn’t Rectangle on the Mac App Store?

### DISCUSSION THREAD

**Discussion by singhalrishi27** [rxhanson/Rectangle#793] [Q&A] [answered]
Bring the app to Mac app store there is no reason not to do it.
I didn't know rectangle exist I used magnet cracked app bringing to MAS will make it easier for new users to discover this app easily...

**Comment by rxhanson** [comment, marked as answer]
Rectangle cannot be listed in the Mac App Store because it cannot be sandboxed. Apps like magnet are in there because they got in ~10 years go before Apple enforced this.

**Reply by ghost** [reply]
I find it odd that Apple allows apps to be updated that no longer fit the App Store criteria.

**Reply by rxhanson** [reply]
I don't :)

Allowing Magnet and others to be updated provides a way for desired functionality that is not in macOS but is in other operating systems to exist without Apple having to do anything. It's a shame that the playing field isn't level, but the choice isn't mine.
